frameworks/JavaScript/ultimate-express/benchmark_config.json

```diff
@@ -0,0 +1,123 @@
+{
+  "framework": "ultimate-express",
+  "tests": [{
+    "default": {
+      "json_url": "/json",
+      "plaintext_url": "/plaintext",
+      "port": 8080,
+      "approach": "Realistic",
+      "classification": "Micro",
+      "framework": "ultimate-express",
+      "language": "JavaScript",
+      "flavor": "NodeJS",
+      "platform": "nodejs",
+      "webserver": "None",
+      "os": "Linux",
+      "database_os": "Linux",
+      "display_name": "ultimate-express",
+      "versus": "nodejs"
+    },
+    "chakra": {
+      "json_url": "/json",
+      "plaintext_url": "/plaintext",
+      "port": 8080,
+      "approach": "Realistic",
+      "classification": "Micro",
+      "framework": "ultimate-express",
+      "language": "JavaScript",
+      "flavor": "node-chakracore",
+      "platform": "nodejs",
+      "webserver": "None",
+      "os": "Linux",
+      "database_os": "Linux",
+      "display_name": "ultimate-express",
+      "versus": "nodejs"
+    },
+    "mongodb": {
+      "db_url": "/mongoose",
+      "query_url": "/mongooseq?queries=",
+      "update_url": "/mongoose-update?queries=",
+      "fortune_url": "/mongoose-fortune",
+      "port": 8080,
+      "approach": "Realistic",
+      "classification": "Micro",
+      "database": "MongoDB",
+      "framework": "ultimate-express",
+      "language": "JavaScript",
+      "flavor": "NodeJS",
+      "platform": "nodejs",
+      "webserver": "None",
+      "orm": "Full",
+      "os": "Linux",
+      "database_os": "Linux",
+      "display_name": "ultimate-express",
```

Comment:
3th Ultimate-express.
Please add in the name to display [normal, charka, mongo, ... ]